sources/packages/web/src/views/aest/StudentApplicationView.vue

```diff
@@ -45,6 +45,10 @@ export default defineComponent({
       type: Number,
       required: true,
     },
+    versionApplicationId: {
+      type: Number,
+      required: true,
```

Comment:
My bad. In the screenshot I shared, I kept the required: true , but the required must be false.